Cargo.toml

```diff
@@ -1,10 +1,21 @@
+[workspace]
+members = [".", "scripts"]
+
 [package]
 name = "commonware-avs-router"
 version = "0.1.0"
 edition = "2024"
 
 [dependencies]
-alloy = { version = "0.12.6" }
+# Workspace dependencies
+alloy = { workspace = true }
+tokio = { workspace = true }
+url = { workspace = true }
+dotenv = { workspace = true }
+serde = { workspace = true }
+commonware-eigenlayer = { workspace = true }
```

Comment:
technical debt, naming here is ambiguous

Reply:
will make a ticket for this. we can't really fix here until we fix the package name of project https://github.com/BreadchainCoop/commonware-avs-network-lookup